src/modm/board/nucleo_l432kc/board.hpp

```diff
@@ -40,6 +40,8 @@ struct SystemClock {
 	static constexpr uint32_t Usart4 = Apb1;
 	static constexpr uint32_t Usart5 = Apb1;
 
+	static constexpr uint32_t Spi1 = Apb2;
+
```

Comment:
Only Spi1?

Reply:
The board.hpp of the L432KC doesn't set any other, so I added the Spi1 only. Looking into the board description I see there are pins for Spi3, so I added that as well.

BTW: The L432KC only provides pins for UART1 and UART2 (which is shared with STLINK).

Reply:
Hm, we really need to generate this.